scipy/optimize/_differentialevolution.py

```diff
@@ -193,6 +194,20 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
 
         .. versionadded:: 1.7.0
 
+    integrality : 1-D array, optional
+        For each decision variable, a boolean value indicating whether the
+        decision variable is constrained to integer values. The array is
+        broadcast to ``(len(x),)``.
+        If any decision variables are constrained to be integral, they will not
+        be changed during polishing.
+        The solution vectors are passed to the objective function as
+        ``func(np.round(x[integrality]))``.
```

Comment:
```suggestion
```
I don't think this is true; this would mean that `func` is passed only the integral values.

I don't think it needs to be corrected, though - just removed. The user already knows that x[integrality] will be integers. 

Maybe the intent was to explain how the usual differential evolution algorithm has been modified to enforce integrality constraint. That makes sense, but I think the notes section would be a more appropriate place.